common:
  frequency:                  10
  topics:
    input:
      observations:           "/AS/P/pcl_filtering/observations"
      map:                    "/map/accumulated"
      odometry:               "/limovelo/state"
      left_detections:        "/camera/left/detections"
      right_detections:       "/camera/right/detections"
    output:
      cones:                  "/AS/P/ccat/cones"

preproc:
  cluster_dist:               0.5

matcher:
  common:
    debug:                    true
    max_match_search_dist:    30.0
    max_match_real_dist:      50.0
    image_width:              1024
    image_height:             768
    cone_height:              0.325
    cone_width:               0.228
  topics:
    output:
      left_projected:         "/AS/P/ccat/projected/left"
      right_projected:        "/AS/P/ccat/projected/right"
      left_pcl:               "/AS/P/ccat/pcl/left"
      right_pcl:              "/AS/P/ccat/pcl/right"

tracker:
  debug:                      true
  same_cone_max_distSq:       0.09
  fancy_markers:              false
  topics:
    output:
      markers_base_link:      "/AS/P/ccat/markers/base_link"
      markersGlobal:          "/AS/P/ccat/markers/global"